zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -748,6 +768,67 @@ where
         }
         .boxed()
     }
+
+    fn get_block_subsidy(&self, height: Option<u32>) -> BoxFuture<Result<BlockSubsidy>> {
+        let latest_chain_tip = self.latest_chain_tip.clone();
+        let network = self.network;
+
+        async move {
+            let height = if let Some(height) = height {
+                Height(height)
+            } else {
+                best_chain_tip_height(&latest_chain_tip)?
+            };
+
+            if height < height_for_first_halving(network) {
+                return Err(Error {
+                    code: ErrorCode::ServerError(0),
+                    message: "Zebra does not support funding stream subsidies, \
+                              use a block height that is after the first halving"
```

Comment:
```suggestion
                    message: "Zebra does not support founders' reward subsidies, \
                              use a block height that is after the first halving"
```

Reply:
Good catch, I'll do this fix in PR #6031.